style.css

```diff
@@ -50,6 +66,7 @@ html {
     --dark-grey: #15181c;
     --grey- : #6e767d;
     --light-grey: #eff3f4;
+    --hover-square-grey: #1c1f23;
```

Comment:
Very nice to store all colors and repeated variables to avoid avoid hex color codes everywhere in the code. Also makes it more readable.

Reply:
I started with HEX because they are shorter but by the end of the project, I found RGBA better, it's easier to change the transparency of it.
Which one do you prefer? HEX or RGBA ?